layouts/default.vue

```diff
@@ -24,6 +36,12 @@ export default {
   computed: {
     isShowBreadcrumb (): boolean {
       return !isLoginRoute()
+    },
+    bannerText (): string | null {
+      const bannerText: string = getFeatureFlag('banner-text')
```

Comment:
It would be a good idea to add the default value of this flag in `src/utils/feature-flags.ts`.

Reply:
Added the default value of 'banner-text' and 'whats-new'.